dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportCLIScriptConfiguration.java

```diff
@@ -55,6 +55,9 @@ public Options getOptions() {
         options.addOption(Option.builder("v").longOpt("validate")
                 .desc("test run - do not actually import items")
                 .hasArg(false).required(false).build());
+        options.addOption(Option.builder("x").longOpt("exclude_content")
```

Comment:
Could we change the long name to `exclude-bitstreams`?  That way it'd exactly match with the `export` script.

Reply:
Done.